src/sql/platform/dialog/browser/dialogModal.ts

```diff
@@ -110,12 +110,12 @@ export class DialogModal extends Modal {
 		dialogButton.hidden ? buttonElement.element.parentElement.classList.add('dialogModal-hidden') : buttonElement.element.parentElement.classList.remove('dialogModal-hidden');
 	}
 
-	protected renderBody(container: HTMLElement): void {
+	protected renderBody(container: HTMLElement): HTMLElement {
```

Comment:
We shouldn't need to return this, the modal class has access to the `container` being passed in here.